app/controllers/api/v1/recordings_controller.rb

```diff
@@ -114,6 +113,18 @@ def recording_params
       def find_recording
         @recording = Recording.find_by! record_id: params[:id]
       end
+
+      def visibility_params_of(visibility)
+        params_of = {
+          Recording::VISIBILITIES[:unpublished] => { publish: false, meta_hash: { protect: false, 'meta_gl3-listed': false } },
```

Comment:
Why `meta_gl3-listed`? We should be using the same value as v2 to allow for backwards compatibility